### ISSUE TITLE
Cannot read property 'kind' of undefined

### ISSUE THREAD

**Issue by Morinohtar** [kirschbaum-development/laravel-translations-loader#11]
In the follow up of the issue #9 i now got this new problem. Don't know really the reason behind it, if it was some module update or what... i haven't done any real change to the code, the same as it was really.

### Error

```sh
$ npm run dev

> @ dev D:\Projects\Laravel\xpto
> npm run development


> @ development D:\Projects\Laravel\xpto
> cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js

 98% after emitting SizeLimitsPlugin ERROR  Failed to compile with 1 errors11:25:42

 error  in ./node_modules/@kirschbaum-development/laravel-translations-loader/all.js

Module build failed (from ./node_modules/@kirschbaum-development/laravel-translations-loader/php.js):
TypeError: Cannot read property 'kind' of undefined
    at parseValue (D:\Projects\Laravel\xpto\node_modules\php-array-parser\index.js:24:15)
    at Object.parse (D:\Projects\Laravel\xpto\node_modules\php-array-parser\index.js:15:10)
    at files.forEach (D:\Projects\Laravel\xpto\node_modules\@kirschbaum-development\laravel-translations-loader\php-loader.js:33:71)
    at Array.forEach (<anonymous>)
    at directories.forEach (D:\Projects\Laravel\xpto\node_modules\@kirschbaum-development\laravel-translations-loader\php-loader.js:24:19)
    at Array.forEach (<anonymous>)
    at Object.execute (D:\Projects\Laravel\xpto\node_modules\@kirschbaum-development\laravel-translations-loader\php-loader.js:18:21)
    at Object.module.exports (D:\Projects\Laravel\xpto\node_modules\@kirschbaum-development\laravel-translations-loader\php.js:25:19)

 @ ./resources/js/backend/administrators/app.js 38:0-157 44:12-26
 @ multi ./resources/js/backend/administrators/app.js ./resources/sass/backend/administrators/app.scss
```

Also ran it inside the VM (homestead), because of the paths and what else, but it gives the same error.

Just to make sure it is only a problem with `Replacing Laravel Parameters` functionality i remove it and then compile it. It work wells in both environments (host and guest).

I even did the thing of creating the `index.js` and put inside the lang folder and add the proper code to the import, but didn't work too.

Btw, a side question: will the package also parse nesting folders inside the different lang folders or just the root files of each lang folder? Didn't confirm this when it was working.

**Comment by luisdalmolin** [comment]
Seems like a problem parsing your translation files. Do you have any special in the PHP files? One thing that is currently a problem is that you can't use PHP constants as noted in #5. If you can't figure out, you can send me the file (replace the values if needed) and I can see from here.

About nesting folders, we currently don't load nesting folders, but I think the package should. I just created issue #12 to handle this.

**Comment by Morinohtar** [comment]
Oh damn, that reminded that i changed one of the translation file to be more dynamic... it was the culprit.
I removed that code and it is back to normal, thanks ;)

About the nesting stuff, that's good.

**Comment by luisdalmolin** [comment]
Sounds good, I'm closing this issue then, but you can watch #12 for the nesting feature.

**Comment by luisdalmolin** [comment]
@Morinohtar Nested folders implemented on version 0.5.0.
